types/aphrodite.d.ts

```diff
@@ -1,6 +1,31 @@
 declare module "aphrodite" {
     import * as React from "react";
 
+    const xs =
+        "@media screen and (max-width: 567px) /* breakpoint.mediaQuery.xs */";
+    const sm =
+        "@media screen and (min-width: 568px) and (max-width: 681px) /* breakpoint.mediaQuery.sm */";
```

Comment:
There should probably be a comment here telling people if they make changes here they also need to update tokens/media-queries.ts.

Reply:
Good call!